schemas/usage.json

```diff
@@ -606,9 +609,8 @@
 					"patternProperties": {
 						"([^\\s]+)": {
 							"$id": "#/usages/library/$template_type",
-							"$comment": "TODO: Should be number",
-							"title": "Count of template-type usage",
-							"type": "string"
+							"$ref": "#/definitions/count_per_post_status",
+							"title": "Libraries per template type"
```

Comment:
Hi, we don't want to override the existing. instead, add the new data in the following format:

```
	"usages": {
		"library": {
			"kit": "1",
			"page": "7",
			"section": "1"
		},
+		"library-details": {
+			"kit": {
+				"draft": 1,
+				"publish": 2,
+			},
+			"page": {
+				"draft": 2,
+				"publish": 5,
+			},
+			"section":  {
+				"publish": 1,
+			},
+		},
```

Reply:
Handled.